aries-site/src/examples/components/dateinput/DateInputSimpleExample.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Box, DateInput } from 'grommet';
+
+export const DateInputSimpleExample = () => {
+  const [value, setValue] = React.useState('');
+  const onChange = event => {
+    const nextValue = event.value;
+    console.log('onChange', nextValue);
+    setValue(nextValue);
+  };
+  return (
+    <Box align="center" pad="large">
+      <DateInput
```

Comment:
It looks like we need a larger drop spacing here (design shows a spacing of xsmall between the input and the drop):
![Screen Shot 2020-08-12 at 11 52 34 AM](https://user-images.githubusercontent.com/1756948/90050406-5f931780-dc93-11ea-9627-73b0bf4bd71d.png)

This should be specified in the theme rather than via props. [I believe] this same drop spacing should be applied across all input drops. This work may make sense to create as a new issue "Audit input drop spacing and apply to theme if necessary." Could you do a brief investigation here to confirm what the best way to proceed is?

Reply:
yes I believe this will also have to do with #657
I can add that to the issue